cluster/examples/kubernetes/kube-registry.yaml

```diff
@@ -40,9 +40,9 @@ spec:
           protocol: TCP
       volumes:
       - name: image-store
-        cephfs:
-          monitors:
-          - INSERT_MONS_HERE
-          user: admin
-          secretRef:
-            name: rook-admin
+        flexVolume:
```

Comment:
do you think this file is even needed since the full content of this yaml is inline in the documentation in https://rook.io/docs/rook/master/k8s-filesystem.html#consume-the-file-system?

Reply:
Right. I was not aware of the existence of this file until today. I rather keep this file and change the docs to link to this file as we do for every other example. @travisn @jbw976 what do you think?

Reply:
we aren't able to directly link to these yaml files from the docs because these yaml files are not versioned along with the docs.  e.g.,  https://rook.io/docs/rook/v0.6/k8s-filesystem.html (note the v0.6 version of the docs) does not have an automated way in our doc generation pipeline to point to a versioned yaml file under `/cluster/examples/kubernetes`.  that's why we have tended to include the content of these files directly into the docs, especially if they are short.

i like having the mysql.yaml in the `/cluster/examples/kubernetes` tree because i use it all the time, but I have a feeling this kube-registry.yaml file is not used nearly as much (if at all) and it could live on only embedded directly into the docs.

Reply:
ok.. should i delete it then?

Reply:
i think that deleting this one would be appropriate since the full content is up to date within the docs, which is the authoritative source.